sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>com.azure.spring</groupId>
   <artifactId>azure-spring-boot-bom</artifactId>
-  <version>3.6.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot-bom;dependency} -->
+  <version>3.5.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot-bom;dependency} -->
```

Comment:
We should not change this.

Reply:
@moarychan , please fix this.

Reply:
@chenrujun

So far this is draft, will not merge.

But you need to fix your version_client.txt or your pom. This change is done by `update_versions.py` automatically whenever an SDK change its version in version_client.txt

Reply:
If you really need 3.6.0, you should write `current` in x-version-update instead of `dependency`. If this 2 lib is not maintained under this repo, one should put them to `external_dependencies.txt`.

Reply:
@weidongxu-microsoft, I got it. Thank you for your information.

Reply:
reverted